content/docs/config.md

```diff
@@ -91,7 +92,7 @@ create-CML-report:
 via a `REPO_TOKEN` variable.
 
 The example above generates visual reports in merge requests:
-[![](/img/GitLab_CML_report.png '=400')](https://gitlab.com/iterative.ai/cml-base-case/-/merge_requests/3)
+![First CML report in GitLab](/img/GitLab_CML_report.png '=400')
```

Comment:
Removed this link as it won't load (to the public at least):

<img width="1039" alt="image" src="https://user-images.githubusercontent.com/1477535/202659240-26bc6f38-6132-4559-9a76-198624e8278f.png">

Reply:
/CC @0x2b3bfa0 please fix :)

Reply:
This 404 seems to still be an issue. Maybe the URL to the MR is different or the GL project needs to be made public?